dali/base/dautils.cpp

```diff
@@ -1991,8 +1992,8 @@ IRemoteConnection *getElementsPaged( IElementsPager *elementsPager,
     if (checkConn && !elem->conn)
         return NULL;
     unsigned n;
-    if (total)
-        *total = elem->totalres.ordinality();
+    if (returned)
+        *returned = elem->totalres.ordinality();
```

Comment:
Only change here is to add the new 'total'. It is used to let the caller know total number of files inside the system. The 'returned' is used by paging. The caller should use it to calculate how many pages (displaying page numbers) can be displayed.